security/pkg/nodeagent/caclient/providers/google/client.go

```diff
@@ -134,6 +135,10 @@ func (cl *googleCAClient) getTLSDialOption() (grpc.DialOption, error) {
 }
 
 func parseZone(clusterURL string) string {
+	// for Hub IDNS, the input is https://gkehub.googleapis.com/projects/HUB_PROJECT_ID/locations/global/memberships/MEMBERSHIP_ID which is global
```

Comment:
It seems like this would already return `""` due to not matching the regex? Or maybe my internal regex compiler is off... Fine to skip early if we know it would fail but want to make sure I understand it

Reply:
Hi John, yeah this is just to clearly skip the Hub case. From the regex compiler and matching point of view, it has the same effect.